libs/net/elastic/Extensions/ServiceCollectionExtensions.cs

```diff
@@ -34,4 +34,28 @@ public static IServiceCollection AddElastic(this IServiceCollection services, IC
 
         return services;
     }
+
+    /// <summary>
+    /// Add a PostgreSQL DbContext to the service collection.
+    /// </summary>
+    /// <param name="services"></param>
+    /// <param name="connectionString"></param>
+    /// <param name="env"></param>
+    /// <returns></returns>
+    /// <exception cref="ArgumentException"></exception>
+    public static IServiceCollection AddElasticSingleton(this IServiceCollection services, IConfiguration config, IHostEnvironment env)
```

Comment:
All services need singletons to manage state between their api and their manager.